CRM/Admin/Form/Setting/Smtp.php

```diff
@@ -170,7 +169,10 @@ public function postProcess() {
         $errorScope = CRM_Core_TemporaryErrorScope::ignoreException();
         $result = $mailer->send($toEmail, $headers, $message);
         unset($errorScope);
-        if (defined('CIVICRM_MAIL_LOG')) {
+        if (defined('CIVICRM_MAIL_LOG_AND_SEND')) {
```

Comment:
Should this be defined('CIVICRM_MAIL_LOG_AND_SEND') && defined('CIVICRM_MAIL_LOG') since per below 
CIVICRM_MAIL_LOG_AND_SEND doesn't work in isolation - also defined is not enough of a test as they could be defined as 0